task15/index.js

```diff
@@ -0,0 +1,92 @@
+const parent = document.getElementById("js-parent"); 
+const modalPlace = document.getElementById('js-modalPlace');
+
+const renderButtonForParent = () => {
+    const button = document.createElement('input');
+    modalPlace.style.display = "none"; 
+    button.id = "js-openBtn";
+    button.type = "button";
+    button.value = "click";
+    parent.appendChild(button);
+};
+renderButtonForParent();
+
+const openBtn = document.getElementById("js-openBtn");
+openBtn.addEventListener("click", (e) => {
+    modalPlace.className = "modal";
+    modalPlace.style.display = "block";
+    e.target.remove();
+    modalClose();
+});
+
+const modalClose = () => {
+    const closeBtn = document.getElementById("js-closeBtn");
+    closeBtn.addEventListener("click", () => {
+        requestValue();
+    });
+};
+
+const requestValue = () => {
+    const getName = document.getElementById("js-getName").value;
+    const getNumber = document.getElementById("js-getNumber").value;
+
+    if( getName.match(/\S/g) && getNumber){
```

Comment:
nits
I was writing same way with you before(``getName.match(/\S/g)``) but I was taught the trim() method and will share it with you.I think it is more readable than regular expressions. (I'll leave it to you to decide if you want to fix it or not!!)

https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/String/trim

Reply:
Very clear! Thanks for teaching me. I fixed it.
dfff462